src/App.tsx

```diff
@@ -92,12 +215,12 @@
               'is-8-desktop',
               'Sidebar',
               {
-                'Sidebar--open': selectedPost,
+                'Sidebar--open': currentPost,
```

Comment:
The className logic for 'Sidebar--open' depends on `currentPost`. Ensure that this logic is correct and that `currentPost` is being set and cleared as expected.